libr/core/cmd_debug.c

```diff
@@ -1651,6 +1651,20 @@ static int cmd_debug_map(RCore *core, const char *input) {
 					if (libname) {
 						char *cmd, *res;
 						const char *file = map->file? map->file: map->name;
+						char *newfile = NULL;
+						if (!r_file_exists (file)) {
+							newfile = r_file_temp ("memlib");
+							if (newfile) {
+#if __WINDOWS__ && !__CYGWIN__
+								const char *nul = "nul";
+#else
+								const char *nul = "/dev/null";
```

Comment:
I remember using null device redirection in more than one place. What about just making it a constant in r_util.h? Will reduce also amount of ifdefs.

Reply:
```
> git grep 'dev/null'| grep c:
binr/radare2/radare2.c:		const char nul[] = "/dev/null";
libr/asm/arch/xtensa/gnu/elf32-xtensa.c:     when building uclibc, which runs "ld -b binary /dev/null".  */
libr/bin/pdb/pdb_downloader.c:	if (r_sys_cmd ("cabextract -v > /dev/null") != 0) {
libr/bin/pdb/pdb_downloader.c:	const char nul[] = "/dev/null";
libr/core/cmd_help.c:	"[cmd] 2> /dev/null", "", "omit the STDERR output of 'cmd'",
libr/lang/p/vala.c:	char *tail = silent?  " > /dev/null 2>&1": "";
```

Reply:
```
> git grep 'dev/null'| grep c:
binr/radare2/radare2.c:		const char nul[] = "/dev/null";
libr/asm/arch/xtensa/gnu/elf32-xtensa.c:     when building uclibc, which runs "ld -b binary /dev/null".  */
libr/bin/pdb/pdb_downloader.c:	if (r_sys_cmd ("cabextract -v > /dev/null") != 0) {
libr/bin/pdb/pdb_downloader.c:	const char nul[] = "/dev/null";
libr/core/cmd_help.c:	"[cmd] 2> /dev/null", "", "omit the STDERR output of 'cmd'",
libr/lang/p/vala.c:	char *tail = silent?  " > /dev/null 2>&1": "";
```

Reply:
can't use char stack arrays if r_sys_devnull() is desired